Extensions/TweenBehavior/tweenruntimebehavior.ts

```diff
@@ -424,10 +426,20 @@ namespace gdjs {
         duration: durationValue,
         easing: easingValue,
         step: function step(state) {
-          // @ts-ignore - objects are duck typed
-          that.owner.setScaleX(state.scaleX);
-          // @ts-ignore - objects are duck typed
-          that.owner.setScaleY(state.scaleY);
+          if(scaleFromCenterOfObject) {
```

Comment:
Nitpicking: to avoid the extra boolean check when running the step function, we could have `originStep` and `centerStep` function, and set the proper one when calling `newTweenable.setConfig`.
Not a huge change,  but can save a very tiny bit on each step function which can run on a lot of objects.

Reply:
I've implemented this slightly differently, changing the body of the function based on the condition. Although it would work (probably better) with named functions, this seemed overkill for how it's being used, but, I'm happy if you want to refactor it!

Reply:
I think this is equivalent and fine 👍